addons/api/.prettierrc.js

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+'use strict';
```

Comment:
None blocking comment: I know this was add by the blueprints, but I forgot if we decide to leave these or delete them.

Reply:
Good question, we have been keeping these changes in other files as well.